src/web/src/components/BannerDynamicItems.tsx

```diff
@@ -1,29 +1,44 @@
-import { makeStyles, Theme } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
+import NoSsr from '@material-ui/core/NoSsr';
+
 import DynamicImage from './DynamicImage';
+import { imageServiceUrl } from '../config';
 
-const useStyles = makeStyles((theme: Theme) => ({
+const useStyles = makeStyles((theme) => ({
   dynamic: {
     height: '100vh',
     transition: 'opacity 1s ease-in-out',
     backgroundColor: theme.palette.primary.main,
     opacity: 0.9,
-    [theme.breakpoints.down(1024)]: {
-      height: 'calc(100vh - 64px)',
-    },
-    [theme.breakpoints.down('xs')]: {
-      height: 'calc(100vh - 56px)',
-    },
+    backgroundSize: 'cover',
+    backgroundPosition: '50% 0',
+    // fallback banner background
+    background: 'linear-gradient(to right, #021B79, #0575E6)',
   },
 }));
 
-const BannerDynamicText = () => {
+// the placeholder and the main image must refer to the same source
+// we use this to request a consistent image vs a random one
+const index = Math.floor(Math.random() * 100 + 1);
```

Comment:
Should we use a larger upper-bound here so that we can access images beyond 100 when we get there?  Your circular buffer should be fine with something like 1000.